### ISSUE TITLE
An unexpected error has ocurred. (UI)

### ISSUE THREAD

**Issue by Shaddy** [TeamCodeStream/CodeStream#28]
When accessing to a channel it crashes and says:

`An unexpected error has ocurred. Click here to reload this tab`

This is the error that seems to produce this.

```
[2018-12-11 13:03:22:197] Exception caught in React component tree: Cannot read property 'references' of undefined — {"info":{"componentStack":"\n    in t\n    in InjectIntl(t)\n    in Connect(InjectIntl(t))\n    in div\n    in t\n    in t\n    in Connect(t)\n    in div\n    in t\n    in div\n    in div\n    in div\n    in div\n    in div\n    in n\n    in InjectIntl(n)\n    in Connect(InjectIntl(n))\n    in Unknown\n    in Connect(Component)\n    in i\n    in IntlProvider\n    in t"},"stacktrace":"TypeError: Cannot read property 'references' of undefined\n    at Array.e.exports (vscode-resource:/c%3A/Users/Sherab/.vscode/extensions/codestream.codestream-0.30.2/dist/webview/webview.js:88:709669)\n    at o.tokenize (vscode-resource:/c%3A/Users/Sherab/.vscode/extensions/codestream.codestream-0.30.2/dist/webview/webview.js:88:706765)\n    at o.parse (vscode-resource:/c%3A/Users/Sherab/.vscode/extensions/codestream.codestream-0.30.2/dist/webview/webview.js:88:706942)\n    at Array.e.exports (vscode-resource:/c%3A/Users/Sherab/.vscode/extensions/codestream.codestream-0.30.2/dist/webview/webview.js:88:685053)\n    at r.process (vscode-resource:/c%3A/Users/Sherab/.vscode/extensions/codestream.codestream-0.30.2/dist/webview/webview.js:88:684501)\n    at v.parse (vscode-resource:/c%3A/Users/Sherab/.vscode/extensions/codestream.codestream-0.30.2/dist/webview/webview.js:88:650247)\n    at e.exports.e.renderer.render.s (vscode-resource:/c%3A/Users/Sherab/.vscode/extensions/codestream.codestream-0.30.2/dist/webview/webview.js:88:755180)\n    at s (vscode-resource:/c%3A/Users/Sherab/.vscode/extensions/codestream.codestream-0.30.2/dist/webview/webview.js:92:355)\n    at Sa (vscode-resource:/c%3A/Users/Sherab/.vscode/extensions/codestream.codestream-0.30.2/dist/webview/webview.js:92:34208)\n    at g (vscode-resource:/c%3A/Users/Sherab/.vscode/extensions/codestream.codestream-0.30.2/dist/webview/webview.js:92:26970)"}
Error: Exception caught in React component tree: Cannot read property 'references' of undefined
```

**Comment by akonwi** [comment]
Hey @Shaddy, is there any markdown in the recent posts of that channel? That error is thrown from the library we're using to parse and render markdown but I can't reproduce it.

**Comment by akonwi** [comment]
Sentry issue: [VSCODE-CODESTREAM-FS](https://sentry.io/codestream-inc/vscode-codestream/issues/803103309/?referrer=github_plugin)

**Comment by Shaddy** [comment]
Hi @akonwi, 

Yes, there are many different items in that channel. They are ``` blocks.

**Comment by akonwi** [comment]
If you could email me (akonwi at codestream) a sample of a couple of those blocks for me to test with, that would be very helpful.

**Comment by eamodio** [comment]
Fixed in 0.30.3
